database/migrations/platform/2024_05_25_000001_update_player_games_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('player_games', function (Blueprint $table) {
+            $table->unsignedInteger('achievements_unlocked_softcore')
+                ->nullable();
```

Comment:
please add an `->after('achievements_unlocked_hardcore')` to put the column near the related existing columns.